docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -1,37 +1,109 @@
-# Onboard public officers
+# Onboard to TechPass as public officers
+
+This article guides you to do the following:
+- [Sign up for TechPass and SEED](#step-1-sign-up-for-techpass-and-seed).
+- [Set up security verification for WOG account](#step-2-set-up-security-verification-for-the-wog-account).
+- [Accept TechPass onboarding invitation](#step-3-accept-techpass-invitation).
+- [Onboard to TechPass account](#step-4-onboard-to-techpass).
+
+<!--TechPass](https://portal.techpass.gov.sg)
+[SEED](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/)-->
+
+You can use the [TechPass portal](http://portal.techpass.gov.sg) to do a self-service sign-up for a TechPass account, or another public officer can invite you to TechPass via the [TechBiz Portal](portal.techbiz.suite.gov.sg). For more information, refer to [TechBiz documentation](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
+
+## Audience
+
+- Public officers using a non-SE GSIB device with a WOG ID. WOG ID is the organisational email address belonging to the *gov.sg* domain and is in the format of *your_name<span>@</span><acronym for your agency>.gov.sg*.
```

Comment:
I worry this statement is misleading
"WOG ID is the organisational email address belonging to the *gov.sg* domain"
not all .gov.sg are wog ids. Eg. hive.gov.sg and open.gov.sg or mindef.gov.sg

Reply:
oh okok. Updated

Reply:
Thank you for spotting this!